[2024-04-23T15:51:52.778+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-23T15:51:52.802+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline.es manual__2024-04-23T15:51:25.785708+00:00 [queued]>
[2024-04-23T15:51:52.924+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline.es manual__2024-04-23T15:51:25.785708+00:00 [queued]>
[2024-04-23T15:51:52.924+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-23T15:51:52.932+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): es> on 2024-04-23 15:51:25.785708+00:00
[2024-04-23T15:51:52.942+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=196) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-23T15:51:52.943+0000] {standard_task_runner.py:63} INFO - Started process 198 to run task
[2024-04-23T15:51:52.941+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline', 'es', 'manual__2024-04-23T15:51:25.785708+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/air_quality/air_quality_dag.py', '--cfg-path', '/tmp/tmpv8we677j']
[2024-04-23T15:51:52.943+0000] {standard_task_runner.py:91} INFO - Job 33: Subtask es
[2024-04-23T15:51:52.975+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline.es manual__2024-04-23T15:51:25.785708+00:00 [running]> on host 1b9b58bb0dc3
[2024-04-23T15:51:53.035+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipeline' AIRFLOW_CTX_TASK_ID='es' AIRFLOW_CTX_EXECUTION_DATE='2024-04-23T15:51:25.785708+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-23T15:51:25.785708+00:00'
[2024-04-23T15:51:53.035+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-23T15:51:53.045+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-23T15:51:53.045+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/air_quality/tasks.py", line 15, in trasform
    df = conn.transform()
         ^^^^^^^^^^^^^^^^
TypeError: ConnectorCity.transform() takes 0 positional arguments but 1 was given
[2024-04-23T15:51:53.053+0000] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=pipeline, task_id=es, execution_date=20240423T155125, start_date=20240423T155152, end_date=20240423T155153
[2024-04-23T15:51:53.060+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 33 for task es (ConnectorCity.transform() takes 0 positional arguments but 1 was given; 198)
[2024-04-23T15:51:53.076+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-04-23T15:51:53.089+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-23T15:51:53.090+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
